framework/src/modules/chain/blocks/blocks.js

```diff
@@ -302,6 +304,26 @@ class Blocks extends EventEmitter {
 
 	// Process a block from the P2P
 	async receiveBlockFromNetwork(block) {
+		this.logger.info(
+			`Received new block from network with id: ${block.id} height: ${
+				block.height
+			} round: ${this.slots.calcRound(
+				block.height
+			)} slot: ${this.slots.getSlotNumber(block.timestamp)} reward: ${
+				block.reward
+			} version: ${block.version}`
+		);
+
+		const receiveBlockImplementations = {
+			1: this.receiveBlockFromNetworkV1,
```

Comment:
I think we have similar idea.
Let's align with team tomorrow how to switch between the codes